server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java

```diff
@@ -51,6 +58,43 @@
                 .body(result);
     }
 
+    @RequestMapping("/rss/fr_FR")
+    @ApiOperation(value = "Articles RSS (fr)", httpMethod = "GET")
```

Comment:
I used the values returned by ctrl.currentLanguage in view, quick & dirty way. We can check that together today.